contrib/test/integration/system.yml

```diff
@@ -57,6 +58,22 @@
   async: 600
   poll: 10
 
+- name: Add python2-boto for Fedora
+  package:
+    name: "{{ item }}"
+    state: present
+  with_items:
+   - python2-boto
+  when: ansible_distribution in ['Fedora']
+
+- name: Add python-boto for RHEL and CentOS
+  package:
+    name: "{{ item }}"
+    state: present
+  with_items:
+   - python-boto
+  when: ansible_distribution in ['RedHat', 'CentOS']
+
```

Comment:
why all these modifications here? where are those needed? right now these will only be picked with /test ami

Reply:
Wait, we don't use this to set up the AMIs?  The package name is different between Fedora and RHEL/CentOS, and this made the task succeed for me.

Reply:
mmm where? I mean, current master is super ok test-wise, where was it failing for you?

Reply:
> Wait, we don't use this to set up the AMIs?

we do but the AMI is cached, so, the job here is not running with this change, yet it pass right?

Reply:
Ah, I was running the whole playbook on a fresh CentOS install.

Reply:
well, I guess there's no harm in doing this as long as /test ami works, let's see